src/common/njord.ts

```diff
@@ -424,13 +425,27 @@ const canAward = async ({
   ctx: AuthContext;
   receiverId?: string | null;
 }): Promise<void> => {
+  if (!receiverId) {
+    throw new ConflictError('Can not award this entity');
+  }
+
   if (ctx.userId === receiverId) {
     throw new ForbiddenError('Can not award yourself');
   }
 
   if (isSpecialUser({ userId: receiverId })) {
     throw new ForbiddenError('Can not award this user');
   }
+
+  if (
+    (await checkCoresAccess({
+      ctx,
+      userId: receiverId,
+      requiredRole: CoresRole.Creator,
+    })) === false
+  ) {
+    throw new ForbiddenError('You can not award this user');
```

Comment:
Is this user facing?

Reply:
Yeah error should propagate, but on frontend side we can show whatever, why do you ask?

Reply:
just because of the copy. if we're not showing this copy, then it's fine

Reply:
I will note down to verify copy with product on frontend side